examples/official-storybook/stories/addon-docs/dynamic-title.stories.js

```diff
@@ -0,0 +1,7 @@
+const getTitle = () => `Addons/Docs/${['dynamic title'][0]}`;
+
+export default {
+  title: getTitle(),
```

Comment:
I personally think this is a bad idea, sure it's expressive, but we're trying to make CSF simple and statically analyzable as much as possible.

@shilman @tmeasday what are your opinion on supporting this?

Does supporting this add complexity?
Is it worth it?
To what degree are dynamic titles ok?

Reply:
I think that dynamic story titles are very common. Especially in big monorepo projects. The proposed PR doesn't directly support this it only fills the gap. I didn't see any docs in storybook about limitations to title, but indeed I could see a section in docs about using babel macros to generate title. So now it looks like feature gap.

Reply:
I think the last discussion we had about it we were tending to lean towards rather than allowing fully dynamic titles to instead to look at common systems of generating them. The primary one is of course setting titles based on filesystem paths -- are there other ways people are setting dynamic titles apart from purely based on the path of the file?

Reply:
We also include package.json name.
But is this PR relevant place to discuss it? 

I understand that you discuss about the approach to generate titles but I don't see any other ways to fix the source code button rather than not relying on final id because looks like it is not necessary.

Reply:
I'm guessing we should merge this and discuss options on what to do with the above